src/main/webapp/gameStage.css

```diff
@@ -0,0 +1,458 @@
+.step-caption {
+  position: absolute;
+  left: 0;
+  top: 20%;
+  width: 100%;
+  text-align: center;
+}
+
+.step-caption spam.step-border {
+  padding: 18px;
+  font-size: 45px;
+}
+
+.step-left {
+    text-align: left;
+}
+
+/** This class allow the text inside it to stay in the container no matter the device width **/
+.step-wrap {
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  hyphens: auto;
+}
+
+.step-align {
+    margin-left: 20%;
+    margin-right: 20%;
+}
+
+/** creators table based on a responsive design **/
+.step-table-align {
+    margin-top: 10px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.step-table-align td {
+    margin-left: 10px;
+    margin-right: 10px;
+}
+
+.step-table-align img {
+    height: 355px;
+    width: 355px;
+    border: 2px solid var(--orange);
+}
+
+.step-figcaption {
+  background-color: black;
+  color: white;
+  font-style: italic;
+  padding: 2px;
+  text-align: center;
+}
+
+/** style for the buttons **/
+.step-button{
+  background-color: var(--orange);
+  border: none;
+  color: white;
+  padding: 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+  border-radius: 8px;
+  font-family: 'Press Start 2P', cursive; 
+}
+
+/** These classes are in charge of styleing the footer **/
+.step-row {
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap; 
+  flex-wrap: wrap;
+  padding: 0 4px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.step-column { 
+  -ms-flex: 32%;
+  flex: 32%;
+  max-width: 32%;
+  padding: 0 4px;
+}
+
+.step-column h4 { 
+  text-align: left;
+}
+
+.resource-link li {
+  list-style-type: none;
+  color: var(--yellow);
+  text-align: left;
+}
+
+.resource-link a:visited {
+  color: var(--yellow);
+}
+
+/** accordian pop up for resources **/
+#accordion {
+  text-align: left;
+  margin-left: 39px;
+}
+
+#accordion section p {
+  display: none;
+}
+
+#accordion section ul {
+  display: none;
+}
+
+#accordion section:target p {
+  display: block;
+}
+
+#accordion section:target ul {
+  display: block;
+}
+
+/** responsive team page **/ 
+.intern-pic {
+  height: 400px;
+  width: 355px;
+  border: 2px solid var(--orange);
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+.step-team-column {
+  float: left;
+  width: 33.3%;
+  margin-bottom: 16px;
+  padding: 0 8px;
+}
+
+.step-team-card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+}
+
+.step-team-container {
+  padding: 0 16px;
+}
+
+.step-team-container::after, .row::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+.step-team-button {
+  border: none;
+  outline: 0;
+  display: inline-block;
+  padding: 8px;
+  color: white;
+  background-color:var(--orange);
+  text-align: center;
+  cursor: pointer;
+  width: 100%;
+}
+
+.step-team-button:hover {
+  background-color: #555; 
+}
+
+/** Responsive media classes to addapt to diffrent devices **/
```

Comment:
typo addapt-->adapt

Reply:
nice catch I had not cough tit in the original style guide